packages/devtools_app/lib/src/shared/diagnostics/references.dart

```diff
@@ -69,7 +69,7 @@ Future<void> addChildReferences(
 
       variable.addAllChildren([
         DartObjectNode.references(
-          'live',
+          'live (application in its current state)',
```

Comment:
wording nit. maybe:

'live (objects currently alive in the application)"
"static (objects alive at the time of snapshot ${selection.heap.snapshotName})"